README.md

```diff
@@ -12,6 +12,23 @@ Ultimately, it aims to support web extensions in a standard, portable,
 cross-platform way. Initially, it will provide a streamlined experience for developing
 [Firefox web extensions](https://developer.mozilla.org/en-US/Add-ons/WebExtensions).
 
+Table of Contents (ToC)
```

Comment:
My request for making this more maintainable remains the same as https://github.com/mozilla/web-ext/pull/1118#discussion_r146373794

Reply:
hey kumar. thanks for the insights in both posts. i will try to fix thing up. cheers